docs/manual/layout/templates/twig/manage.en.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Manage templates'
+description: 'The management of the template files.'
+aliases:
+    - /en/layout/templates/twig/manage-template/
+weight: 20
+---
+
+
+Twig templates are marked differently compared to the previous PHP templates: (`.html.twig` instead of `.html5`).
+
```

Comment:
Do we need this? Because technically they do not need any extension, we're just going to ignore anything but `.twig` in our loader.

```suggestion
```